packages/help-center/src/components/help-center-support-chat-message.tsx

```diff
@@ -100,11 +104,20 @@ export const HelpCenterSupportChatMessage = ( {
 				}
 			} }
 			className={ clsx( 'help-center-support-chat__conversation-container', {
+				'is-unread-message': hasUnread,
 				[ `is-${ conversationStatus }` ]: conversationStatus,
 			} ) }
 		>
-			<div className={ clsx( 'help-center-support-chat__conversation-avatar' ) }>
+			<div
+				className={ clsx( 'help-center-support-chat__conversation-avatar', {
+					'has-unread-messages': unreadMessages > 0,
+				} ) }
+			>
 				{ renderAvatar() }
+
+				{ unreadMessages > 0 && (
+					<div className="help-center-support-chat__conversation-badge">+{ unreadMessages }</div>
+				) }
```

Comment:
What about using the _boolean_ variable instead of duplicating code 😅?

```suggestion
			<div
				className={ clsx( 'help-center-support-chat__conversation-avatar', {
					'has-unread-messages': hasUnreadMessages,
				} ) }
			>
				{ renderAvatar() }

				{ hasUnreadMessages && (
					<div className="help-center-support-chat__conversation-badge">+{ unreadMessages }</div>
				) }
```

Reply:
Oops :)